src/import/load-costume.js

```diff
@@ -261,7 +260,8 @@ const loadCostumeFromAsset = function (costume, runtime, optVersion) {
     }
     if (costume.asset.assetType.runtimeFormat === AssetType.ImageVector.runtimeFormat) {
         return loadVector_(costume, runtime, rotationCenter, optVersion)
-            .catch(() => {
+            .catch(error => {
+                log.warn(`${error.name}: ${error.message}`);
```

Comment:
Minor comment: maybe we should add an error message prefix so it's clear what part of the code caught the error
```suggestion
                log.warn(`Error loading vector image: ${error.name}: ${error.message}`);
```